app/models/ability.rb

```diff
@@ -93,7 +94,9 @@ def signed_in(user)
     can [:index, :show], PhysicalTicket, user: user
 
     can [:new, :create], Booth do |booth|
-      booth.new_record? && booth.conference.program.cfps.for_booths.try(:open?)
+      invited_user = booth.conference.invites.where('end_date >= ?', Date.today).where(
+        user_id: user.id, invite_for: (I18n.t 'booth').capitalize.to_s)
+      invited_user.exists? || booth.new_record? && booth.conference.program.cfps.for_booths.try(:open?)
```

Comment:
I'd use some parenthesis here to make sure it's always obvious what's the order of the logical execution here

Reply:
Alright :+1: